state-chain/runtime/src/lib.rs

```diff
@@ -924,7 +924,7 @@ impl pallet_cf_broadcast::Config<Instance4> for Runtime {
 	type EnsureThresholdSigned =
 		pallet_cf_threshold_signature::EnsureThresholdSigned<Self, EvmInstance>;
 	type BroadcastReadyProvider = BroadcastReadyProvider;
-	type BroadcastTimeout = ConstU32<{ 90 * MINUTES }>;
+	type BroadcastTimeout = ConstU32<{ 2 * MINUTES }>;
```

Comment:
Not sure about this - the EVM retrier is the same for ETH and ARB chains. Should the protocol timeout be the same too? 🤔 

Thoughts?

Reply:
Protocol timeout should be longer for ETH, since we have to wait for safety margin